css-layout-classes/styles.css

```diff
@@ -0,0 +1,39 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
+  font-family: Roboto, sans-serif;
+}
+
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+}
+
+h1 {
+  font-size: 4rem;
+}
+
+h2 {
+  font-size: 2.5rem;
+}
+
+img {
+  height: 28rem;
+}
```

Comment:
You don't want to set a `height` on the `img` you just want to change it's `width`. We want it to take up `100%` of it's parent